riptide-failsafe/README.md

```diff
@@ -133,6 +133,9 @@ Http.builder().requestFactory(new HttpComponentsClientHttpRequestFactory())
 If no executor is specified, the default executor configured by `Failsafe` is used. See [Failsafe DelegatingScheduler class](https://github.com/failsafe-lib/failsafe/blob/master/core/src/main/java/dev/failsafe/internal/util/DelegatingScheduler.java#L111), 
 and also [Failsafe documentation](https://failsafe.dev/async-execution/#executorservice-configuration) for more information.
 
+**Beware** when specifying a custom `ExecutorService` - the `ExecutorService` should have a core pool size or parallelism 
```

Comment:
Didn't @fatroom also mention something about mentioning to not use the same `ExecutorService` instance for everything?

Reply:
I can't see it, but I can add this recommendation as well

Reply:
To quote from @fatroom `user shared the same ExecutorService between several Plugins or Http clients`.

Reply:
But maybe he meant something different here…